lib/Data/Chronicle/Reader.pm

```diff
@@ -130,7 +130,7 @@ sub get {
 
     if (blessed($self->cache_reader)) {
         my $cached_data = $self->cache_reader->get($key);
-        return JSON::from_json($cached_data) if defined $cached_data;
+        return decode_json($cached_data) if defined $cached_data;
```

Comment:
where `decode_json` is taken from? I don't see any defined nor imported function. Better to use explicitly `JSON::MaybeXS->new->decode()`,

Reply:
It's a default import from `JSON` but yes, I'd prefer to have explicit JSON and UTF-8 decoding here.

Reply:
Code is updated with `encode_utf8` in mind and migration to `JSON::MaybeXS`